packages/core/src/sass/app/_common/components/cookie-banner.scss

```diff
@@ -0,0 +1,96 @@
+/** @define animation */
+@keyframes FadeInDown {
```

Comment:
Why can't we just use `transform` via `translate` and `opacity` for the `fadeInDown` and `fadeOutUp` effects instead keyframes? 

`transforms` are more performant than using `top` or `bottom` (which renders every frame).

Reply:
Applied.